docs/UserGuide.md

```diff
@@ -531,4 +589,18 @@ This glossary provides definitions for the special terms used in this user guide
 
 **URL:** The address of a web page.
 
+## 8. **Appendix**
+
+### Sample Data
+
+#### Sample Data (Students)
+
+![sample_student_1](images/enhao/sample1.PNG)
+![sample_student_2](images/enhao/sample2.PNG)
+![sample_student_3](images/enhao/sample3.PNG)
+
+#### Sample Data (Sessions)
+
+![sample_session_1](images/enhao/sample_ses1.PNG)
+![sample_session_2](images/enhao/sample_ses2.PNG)
```

Comment:
I realise when viewing it, the images are of different size, so it's jutting out abit. I can help you combine them into one image so it looks nicer?

Reply:
Yeap sure! I think that might be better! Thanks! 🥇